scarlet/source.py

```diff
@@ -44,7 +46,7 @@ def __init__(self, model_frame, observations=None):
         if observations is None:
             spectrum = np.random.rand(C)
         else:
-            spectrum = get_best_fit_seds(image[None], observations)[0]
+            spectrum = get_best_fit_spectra(image[None], observations)[0]
```

Comment:
Ha! I wanted to change these in my PR, but realising that it crashed the tests, I renounced.